templates/cli/lib/commands/generic.js.twig

```diff
@@ -77,19 +77,19 @@ const client = new Command("client")
             }
         }
 
-        if (key) {
+        if (key !== undefined) {
             globalConfig.setKey(key)
         }
 
-        if (projectId) {
+        if (projectId !== undefined) {
             localConfig.setProject(projectId, '');
```

Comment:
What is the second parameter used for?

Reply:
setProject(projectId, projectName)
    Its the project name